web-app-router/components/AlertIconContainer/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { ReactNode } from 'react'
+
+type AlertIconContainerProps = {
+  children: ReactNode
+}
+
+export const AlertIconContainer = (props: AlertIconContainerProps) => {
```

Comment:
@michaelfurutin as I can see from your screenshot it's possible to use this container with different color, maybe it worth to name it something like `DialogStatusIconContainer`

Reply:
If we look at [this screen](https://www.figma.com/file/URO9Y5UPOqK3LbLJ9623Gv/%F0%9F%AA%AA-World-ID-2.0?type=design&node-id=1668-34698&mode=dev), we'll see that this component can be used independently of the dialog.

I tried to create a common component and set its color through props, but I couldn't get it to work. There was some red shade left in the gradient that I couldn't overcome. We need to dive in and investigate. I can try again.

Reply:
I have an Idea, let me try